src/tables/sortable-table.js

```diff
@@ -36,37 +36,89 @@ import classnames from 'classnames'
 /* eslint react/jsx-key: 0 */
 
 const propTypes = {
+  className: PropTypes.string,
   columns: PropTypes.arrayOf(Types.column).isRequired,
-  controlled: PropTypes.bool.isRequired,
   data: PropTypes.arrayOf(PropTypes.object).isRequired,
+  initialSortPath: PropTypes.string.isRequired,
+  initialSortFunc: PropTypes.func,
+  initialAscending: PropTypes.bool,
+  disableReverse: PropTypes.bool,
   disableSort: PropTypes.bool.isRequired,
+  controlled: PropTypes.bool,
+  onChange: PropTypes.func,
   rowComponent: Types.component,
   headerComponent: Types.component,
-  className: PropTypes.string,
-  ...sortablePropTypes,
 }
-
 const defaultProps = {
-  className: ''
+  className: '',
+  initialAscending: true,
```

Comment:
Let's add a default prop for every non-required prop, even if it's just `null`.

Reply:
I did this except for the header/row component definitions. `null` is a valid component definition, which caused all of the tests to break. I could explicitly initialize to `undefined` but that would seem to default the purpose.

Reply:
Ah, I guess that's what I get for using [destructuring defaults](https://zaiste.net/posts/javascript-destructuring-assignment-default-values/#:~:text=Default%20values%20in%20destructuring%20assignement,values%20in%20the%20destructuring%20statement.&text=You%20can%20combine%20default%20values%20with%20renaming%20in%20the%20destructuring%20assignment.) too heavily. Fine to leave it for now.